src/utils/expressions.js

```diff
@@ -20,3 +22,12 @@ export function wrapExpression(input: string) {
     }
   \`)`.trim();
 }
+
+export function isInvalidExpression(input: string) {
```

Comment:
i think `validateSyntax` reads a bit better, i still expect a boolean return value. thoughts?

Reply:
cool! ... wondering if `validateSyntax`  returning a boolean won't be confusing. Maybe we should return the expression string if it doesn't fail and if it fails we return the error object.

Reply:
I see, it might be nice to show a formatted error object "Syntax Error: foo)( is terrible`.

yep. up to you. I don't know if you can serialize `e` and send it down from the worker, but something simple could work. Success `true` error, `{error: {message: "YO", name: "SyntaxError" }}`